contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeValidator.vue

```diff
@@ -32,10 +36,14 @@
       node: {
         type: Object,
       },
+      dot: {
+        type: Boolean,
+        default: false,
+      },
     },
     computed: {
       noTitle() {
-        return !this.node.title;
+        return !this.dot && !this.node.title;
```

Comment:
Why is `dot` property part of this condition? If it's related only to UI I would suggest moving to the template outside of `noTitle` computed property.

Reply:
I had originally thought of this as  "icon only," but this didn't make sense as we do show error icons. This is more of a "dot mode" where only a dot is shown